install.sh

```diff
@@ -3,3 +3,6 @@
 set -e
 
 sudo apt-get install -y curl awscli
+
+ln -s ${GEOCINT_WORK_DIRECTORY}/completeness_report/ ${GEOCINT_WORK_DIRECTORY}/public_html/completeness_report/
```

Comment:
Also, it seems that `${GEOCINT_WORK_DIRECTORY}/completeness_report/ `shoud be rather `${GEOCINT_WORK_DIRECTORY}/**geocint**/completeness_report/`

Reply:
on Stratum both works
https://stratum.kontur.io/geocint/report/completeness.html
https://stratum.kontur.io/report/completeness.html
I think no need to use /geocint?

Reply:
changed folder structure